### ISSUE TITLE
Image size dimensions

### ISSUE THREAD

**Issue by ctlui** [sathish4fri/verified-account#4]
Hi Sathish,

Not an issue but 'speed' improvement:
https://gtmetrix.com/specify-image-dimensions.html

https://github.com/sathish4fri/verified-account/blob/master/plugins/default/profile/pages/profile.php#L85
margin-bottom: -5px;" src="<?php echo ossn_site_url("components/verified-account/images/verified-admin.png") ?>" height="25" width="25"/>

https://github.com/sathish4fri/verified-account/blob/master/plugins/default/profile/pages/profile.php#L90
margin-bottom: -5px;" src="<?php echo ossn_site_url("components/verified-account/images/verified.png") ?>" height="25" width="25"/>

Thanks - Eric

**Comment by sathish4fri** [comment]
this images wont reduce that much speed .

**Comment by ctlui** [comment]
Yes I know, but the problem is this 2 + 2 + 2 +2 etc...and he have yet many like that in Ossn
